CONTRIBUTING.md

```diff
@@ -224,6 +225,54 @@ GitHub has a [nice introduction](https://help.github.com/articles/github-flow/)
 <img align="center" src="https://i.imgur.com/s8yELfK.png" alt="Example-Contribution" width="800"/>
 
 
+## Updating the schema
+
+Portions of the BIDS specification are defined using YAML files, in order to make the specification machine-readable.
+Currently, the only portion of the specification that relies on this schema is the Entity Table, but any changes to the specification should be mirrored in the schema.
+
+### The format of the schema
+
+The schema reflects the files and objects in the specification, as well as associations between these objects.
+Here is a list of the files and subfolders of the schema, with a description for each:
+
+- `associated_data.yaml`:
+    Folders that contain files falling outside the jurisdiction of the specification, but which are commonly contained within the same folder as a BIDS dataset.
+    The schema specifies which folders are accepted and whether they are required or optional.
```

Comment:
I might just be being dense here, but are are folders "outside the jurisdiction" of the spec but which still can be considered "required"?  Isn't that a contradiction?

Reply:
That's a good point. I included a `required` field for the items in this file mostly to match my convention in other files, but we can assume anything in the file is not required. Rather than abandon that right now, I'll try to make the text a little less confusing. If that doesn't resolve it, I can drop the `required` field in the file.